docs/getting-started/faq.md

```diff
@@ -74,7 +74,7 @@ Gutenberg is developed on [GitHub](https://github.com/WordPress/gutenberg) under
 There are four phases of Gutenberg which you can see on the [official WordPress roadmap](https://wordpress.org/about/roadmap/). As of writing this, we’re currently in phase 2:
 
 1. Easier Editing — Already available in WordPress since 5.0, with ongoing improvements.
-2. Customization — Full Site editing, Block Patterns, Block Directory, Block based themes.
+2. Customization — Site Editing, Patterns, Block Directory, Block based themes.
```

Comment:
Let's keep the changes specific to reducing "Block Pattern", not other minor tweaks please. 🙇

Reply:
Thank you, @richtabor! Done!